cam0_roi: [0, 0, 768, 678]
cam1_roi: [0, 0, 768, 960]
cam2_roi: [0, 0, 768, 906]
cam3_roi: [0, 0, 768, 860]
cam4_roi: [0, 0, 768, 845]
cam5_roi: [0, 0, 768, 1024]